enos/k8s/enos-terraform-k8s.hcl

```diff
@@ -0,0 +1,20 @@
+terraform "k8s" {
```

Comment:
Any reason why we don't call this default since there's only a single terraform config?